config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -21,6 +21,26 @@ presubmits:
         # /home/prow/go/src/github.com/{{.Org}}/{{.Repo}}, so just '.' should
         # suffice, but it's nice to be explicit.
         - -thin-manifest-dir=/home/prow/go/src/github.com/kubernetes/k8s.io/k8s.gcr.io
+  # Check that images to be promoted are free of vulnerabilities
+  - name: pull-k8sio-cip-vuln
+    annotations:
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: wg-k8s-infra-k8sio
+    decorate: true
+    skip_report: false
+    run_if_changed: 'k8s.gcr.io/((images/.*/images\.yaml)|(manifests/.*/promoter-manifest\.yaml))'
+    max_concurrency: 10
+    branches:
+    - ^master$
+    spec:
+      serviceAccountName: k8s-infra-gcr-promoter-vuln-scanning
```

Comment:
/cc @fejta 
I don't think we allow presubmits to run as trusted service accounts that have special access.  Could you explain more about what this job is doing?

Reply:
What cluster should this pod run in? Where is the k8s-infra-gcr-promoter-vuln-scanning service account defined?

Reply:
@spiffxp this presubmit will check new images to be promoted for any vulnerabilities by looking up the image through the Container Analysis service. This is why the Container Analysis API was enabled on all of the image staging projects last week.

Reply:
The kubernetes service account is defined for the (default) build cluster here: https://github.com/kubernetes/test-infra/blob/master/config/prow/cluster/build_serviceaccounts.yaml#L21-L28

The google service account is defined here: https://github.com/kubernetes/k8s.io/blob/master/infra/gcp/ensure-prod-storage.sh#L346-L350

It's given a role for each k8s-staging-* project in the kubernetes.io org here:
https://github.com/kubernetes/k8s.io/blob/4adcbd780a71d502a0d8fceb456f57ef7e7353af/infra/gcp/ensure-staging-storage.sh#L150-L153

That role appears to be `roles/containeranalysis.occurrences.viewer`: https://github.com/kubernetes/k8s.io/blob/8d556981bb18d33d9353d7b350b71794a0ae2021/infra/gcp/lib.sh#L186-L201

Which is defined here https://cloud.google.com/iam/docs/understanding-roles#container-analysis-roles

It's still not clear to me whether this is something I would want to allow any job from the public to run as? But it doesn't expose any destructive operations at least